.gitignore

```diff
@@ -0,0 +1,8 @@
+# dependencies
+/node_modules
+
+# misc
```

Comment:
I wonder if we should add this in this repo, that should be set in your global `.gitignore`, no?

Reply:
That's normally what I do, but I figure that since this is public it will be less trouble if somehow we get an external contribution.